apps/jsonrpc2/lib/jsonrpc2/spec_handler.ex

```diff
@@ -1,5 +1,114 @@
+  def handle_request("eth_getStorageAt", _), do: {:error, :not_supported}
+  def handle_request("eth_getTransactionCount", _), do: {:error, :not_supported}
+  def handle_request("eth_getBlockTransactionCountByHash", _), do: {:error, :not_supported}
+  def handle_request("eth_getBlockTransactionCountByNumber", _), do: {:error, :not_supported}
+  def handle_request("eth_getUncleCountByBlockHash", _), do: {:error, :not_supported}
+  def handle_request("eth_getUncleCountByBlockNumber", _), do: {:error, :not_supported}
+  def handle_request("eth_getCode", _), do: {:error, :not_supported}
+  def handle_request("eth_sign", _), do: {:error, :not_supported}
+  def handle_request("eth_sendTransaction", _), do: {:error, :not_supported}
+  def handle_request("eth_sendRawTransaction", _), do: {:error, :not_supported}
+  def handle_request("eth_call", _), do: {:error, :not_supported}
+  def handle_request("eth_estimateGas", _), do: {:error, :not_supported}
+  def handle_request("eth_getBlockByHash", _), do: {:error, :not_supported}
+  def handle_request("eth_getBlockByNumber", _), do: {:error, :not_supported}
+  def handle_request("eth_getTransactionByHash", _), do: {:error, :not_supported}
+  def handle_request("eth_getTransactionByBlockHashAndIndex", _), do: {:error, :not_supported}
+  def handle_request("eth_getTransactionByBlockNumberAndIndex", _), do: {:error, :not_supported}
+  def handle_request("eth_getTransactionReceipt", _), do: {:error, :not_supported}
+  def handle_request("eth_getUncleByBlockHashAndIndex", _), do: {:error, :not_supported}
+  def handle_request("eth_getUncleByBlockNumberAndIndex", _), do: {:error, :not_supported}
+  # eth_getCompilers is deprecated
+  def handle_request("eth_getCompilers", _), do: {:error, :not_supported}
+  # eth_compileLLL is deprecated
+  def handle_request("eth_compileLLL", _), do: {:error, :not_supported}
+  # eth_compileSolidity is deprecated
+  def handle_request("eth_compileSolidity", _), do: {:error, :not_supported}
+  # eth_compileSerpent is deprecated
+  def handle_request("eth_compileSerpent", _), do: {:error, :not_supported}
+  def handle_request("eth_newFilter", _), do: {:error, :not_supported}
+  def handle_request("eth_newBlockFilter", _), do: {:error, :not_supported}
+  def handle_request("eth_newPendingTransactionFilter", _), do: {:error, :not_supported}
+  def handle_request("eth_uninstallFilter", _), do: {:error, :not_supported}
+  def handle_request("eth_getFilterChanges", _), do: {:error, :not_supported}
+  def handle_request("eth_getFilterLogs", _), do: {:error, :not_supported}
+  def handle_request("eth_getLogs", _), do: {:error, :not_supported}
+  def handle_request("eth_getWork", _), do: {:error, :not_supported}
+  def handle_request("eth_submitWork", _), do: {:error, :not_supported}
+  def handle_request("eth_submitHashrate", _), do: {:error, :not_supported}
+  def handle_request("eth_getProof", _), do: {:error, :not_supported}
+  # db_putString is deprecated
```

Comment:
Does this mean that we'll keep `db_putString` returning `not_supported` forever?

Reply:
Depends on when we release the 1.0 version :) and if they remove it by then...